pkgs/top-level/all-packages.nix

```diff
@@ -3313,7 +3313,8 @@ in
 
   libceph = ceph.lib;
   inherit (callPackages ../tools/filesystems/ceph {
-    boost = boost172.override { enablePython = true; python = python38; };
+    boost = boost175.override { enablePython = true; python = python3; };
```

Comment:
maybe use `boost17x` here ?

Reply:
@nagy incorporated both your suggestions just now, i.e. removed `fetchpatch` and changed to `boost17x`.